### PR TITLE
Load TF model using val-loader

### PR DESCRIPTION
**Pull request by mlaco**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "nsfwjs",
   "version": "0.0.3",
   "description": "Detect NSFW content client side",
-  "main": "dist/index.js",
```
Comment (GantMan) [inline package.json]: Why did you remove `main` here?

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+  "private": true,
```
Comment (GantMan) [inline package.json]:
```
Finally, if you do not wish to grant others the right to use a private or unpublished package under any terms:

{ "license": "UNLICENSED" }
Consider also setting "private": true to prevent accidental publication.
```
Why would I set private to true?

File: package.json [hunk 3]
```diff
@@ -39,5 +39,11 @@
   "bugs": {
     "url": "https://github.com/infinitered/nsfwjs/issues"
   },
-  "homepage": "https://github.com/infinitered/nsfwjs#readme"
+  "homepage": "https://github.com/infinitered/nsfwjs#readme",
+  "dependencies": {
+    "ts-loader": "^5.3.3",
```
Comment (GantMan) [inline package.json]: Does `ts-loader` need to be a dependency or a dev dependency?

File: src/model-loader.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import * as nsfwjs from "./index";
+
+module.exports = () => {
+  let val: { model: any } = { model: undefined };
+  nsfwjs.load("../lib/model").then(_model => {
+    val.model = _model;
+  });
+  return {
+    code: "module.exports = " + val
```
Comment (GantMan) [inline src/model-loader.ts]: this feels so meta

**Comment by GantMan** [comment]
@mlaco - how's this one going?

**Comment by mlaco** [comment]
I'll try to work on it soon
